adapters/revcontent/revcontent.go

```diff
@@ -0,0 +1,91 @@
+package revcontent
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"net/http"
+)
+
+type Adapter struct {
```

Comment:
This doesn't need to be exported and should therefore start with a lower case letter. Please rename to `adapter`.

Reply:
sorry, not sure how that got missed. Fixing.